test-suites/configurator.spec.ts

```diff
@@ -1075,4 +1075,29 @@ makeSuite('PoolConfigurator', (testEnv: TestEnv) => {
     const { helpersContract } = testEnv;
     expect(await helpersContract.getDebtCeilingDecimals()).to.be.eq(2);
   });
+
+  it('Check that the reserves have flashloans enabled', async () => {
+    const { weth, aave, usdc, dai, helpersContract } = testEnv;
+
+    const wethFlashLoanEnabled = await helpersContract.getFlashLoanEnabled(weth.address);
+    expect(wethFlashLoanEnabled).to.be.equal(true);
```

Comment:
why existing assets will have flashloan enabled if the value is 0?

Reply:
I updated the deploy repo to set all reserves as flashloanable. If you delete your node_modules and re-install, the v3-deploy dependency will be brought in and this will pass.

Reply:
Great, just to note that introducing this change in an ongoing market implies enabling flashloan for all existing reserves

Reply:
I can't figure it out

Reply:
Agreed